static/dtale/Filter.jsx

```diff
@@ -103,6 +104,7 @@ class Filter extends React.Component {
               </ul>
             </div>
           </div>
+          <div className="row">{this.props.visible ? <ContextVariables dataId={this.props.dataId} /> : null}</div>
```

Comment:
Yo udon't need the check on `this.props.visible` since it is already checking that at the beginning of the `render()` function